README.md

```diff
@@ -347,6 +348,59 @@ TotalCurrentRevenue.calculate
 AverageDailyRevenue.result
 ```
 
+## Rule and Ruleset
+
+### When to use it
+
+Rule objects provide a place for conditional logic
+
+### Assumptions and rules
+
+* Rule has `#satisfied?`, `#applicable?` and `#forceable?` methods
+* Custom rule has to implement at least `#satisfied?` method
+* Rule requires a subject as first argument
+* Multiple rules can be joined into `Ruleset`
+
+### Examples
+
+#### Declaration
+
+```ruby
+class RoomIsAssignedRule < Rule
+  def satisfied?
+    subject.unit.present?
+  end
+
+  def forceable?
+    false
+  end
+end
+
+class RoomIsCleanRule < Rule
+  def satisfied?
+    !subject.unit.dirty?
+  end
+
+  def not_applicable?
```

Comment:
Line 359 says that there's a `applicable?` method, but here is `not_applicable?`. Personally I'd stick to one of them to avoid confusion

Reply:
actually both make sense - on many occasions you want to see for non-applicable rules or applicable rules - it is like "reject" and "select"... 

Usually you want to **describe** when the rule is NOT applicable (because this sounds like an exception) and **test/query** if a rule is applicable.

Anyway - this should be tackled in documentation section IMO. Together with concept of strong rulesets

Reply:
@stevo Agree

Reply:
@polakowski

Addressed